shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -164,11 +164,43 @@ - (BOOL)isAccessibilityElement {
   // Note: hit detection will only apply to elements that report
   // -isAccessibilityElement of YES. The framework will continue scanning the
   // entire element tree looking for such a hit.
+  if ([self node].HasFlag(blink::SemanticsFlags::kIsEdge))
+    return false;
   return [self node].flags != 0 || ![self node].label.empty() || ![self node].value.empty() ||
          ![self node].hint.empty() ||
          ([self node].actions & ~blink::kScrollableSemanticsActions) != 0;
 }
 
+- (void)collectEdges:(NSMutableArray<SemanticsObject*>*)edges {
+  if ([self node].HasFlag(blink::SemanticsFlags::kIsEdge))
+    [edges addObject:self];
+  if ([self hasChildren]) {
+    for (SemanticsObject* child in self.children) {
+      [child collectEdges:edges];
+    }
+  }
+}
+
+- (NSString*)routeName {
+  NSString* name = nil;
+  if ([self hasChildren]) {
+    for (int i = [self.children count] - 1; i >= 0; i--) {
+      SemanticsObject* child = [self.children objectAtIndex:i];
+      NSString* newName = [child routeName];
+      if (newName != nil && [newName length] > 0) {
+        name = newName;
+      }
+    }
+  }
+  if ([self node].HasFlag(blink::SemanticsFlags::kIsRoute)) {
```

Comment:
Check this first before doing the loop above?

Reply:
Fixed